spec/scrabble_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'spec_helper'
+require_relative '../lib/scrabble.rb'
```

Comment:
You probably want to set up your editor to use 2-space "soft tabs" (which means that the tab key inserts two spaces instead of an actual tab character) - I'm not sure exactly how you do this with Atom but maybe one of your classmates or instructors does.